canonical/snapcraft.io
# WD-720 display correct store on the dispute page

## Done
- Fix wrong store displayed on dispute page
- It shows the store selected by user now
## How to QA
- Go to /register-snap
- Select a non-global store
- Select a reserved snap name
- Submit, this should display the error containing the link to file a dispute
- Click on `file a dispute`
- The dispute page should show the correct store selected by the user
## Issue / Card
Fixes #4151 
Fixes https://warthogs.atlassian.net/browse/WD-720

## Screenshots
### Before
<img width="1290" alt="image" src="https://user-images.githubusercontent.com/47438585/205647308-c7b2bf60-c8b5-4850-86ad-f24b460632e0.png">


### After
<img width="1069" alt="image" src="https://user-images.githubusercontent.com/47438585/205647189-31c88808-6ff0-42d5-95b1-3e9ace74a37a.png">
